cms/static/js/models/group_configuration.js

```diff
@@ -45,7 +46,7 @@ function(Backbone, _, str, gettext, GroupModel, GroupCollection) {
         },
 
         reset: function() {
-            this.set(this._originalAttributes, { parse: true });
+            this.set(this._originalAttributes, { parse: true, validate: true });
```

Comment:
Note to reviewers: this fixes an existing unreported bug where editing a content group/group configuration in an invalid way, clicking cancel, and then re-editing the group/group configuration still displayed the old validation error.